opentracing-api/src/main/java/io/opentracing/ActiveSpan.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2016-2017 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing;
+
+import java.io.Closeable;
+
+/**
+ * In any execution context (or any thread, etc), there is at most one "active" {@link Span}/{@link ActiveSpan}
+ * primarily responsible for the work accomplished by the surrounding application code. That @{link ActiveSpan} may be
+ * accessed via the {@link ActiveSpanSource#activeSpan()} method. If the application needs to capture work that should be
+ * part of the same Span, the Source provides a {@link ActiveSpan#capture} method that returns a {@link Continuation};
+ * this continuation may be used to re-activate and continue the {@link Span} in that other asynchronous executor
+ * and/or thread.
+ *
+ * <p>
+ * {@link ActiveSpan}s are created via {@link Tracer.SpanBuilder#startActive()} or {@link ActiveSpanSource#makeActive}. They
+ * can be {@link ActiveSpan#capture()}ed as {@link ActiveSpan.Continuation}s, then re-{@link Continuation#activate()}d
+ * later.
+ *
+ * @see ActiveSpanSource
+ */
+public interface ActiveSpan extends Closeable, Span {
```

Comment:
Maybe explicitly override the `close()` method to:

- Indicate that no `IOException` must be thrown from close()
- Document that close is intended as a try-with-resources usable alias for `deactivate()`.

(the first is more important than the second bullit)